.travis.yml

```diff
@@ -18,17 +19,13 @@ matrix:
         - os: osx
           env: QT5=True
 before_install:
-   - . ${TRAVIS_BUILD_DIR}/.travis/${TRAVIS_OS_NAME}.${TARGET_OS}.before_install.sh
+    - . ${TRAVIS_BUILD_DIR}/.travis/before_install.sh
```

Comment:
Do you know how to [set permissions](http://stackoverflow.com/questions/21691202/how-to-create-file-execute-mode-permissions-in-git-on-windows) in the repository or not?